pom.xml

```diff
@@ -63,6 +63,7 @@
         <jettison.version>1.5.1</jettison.version>
         <woodstox.version>5.4.0</woodstox.version>
         <checkstyle.suppressions.location>checkstyle/suppressions.xml</checkstyle.suppressions.location>
+        <com.nimbusds.nimbus-jose-jwt.version>9.29</com.nimbusds.nimbus-jose-jwt.version>
```

Comment:
is this dependency needed? can it be excluded?

Reply:
it looks unlikely that hadoop-auth is in use.

Reply:
Let me try excluding it...

Reply:
Could successfully build and test the connector after excluding nimbus-jose-jwt